content/logs/log_collection/docker.md

```diff
@@ -27,118 +27,122 @@ Two installations are possible:
+Collecting logs is *disabled* by default in the Datadog Agent. Add the following to it in `datadog.yaml`: 
 
-Then, mount the directory by adding `-v /opt/datadog-agent/conf.d:/conf.d:ro` in the run command of the containerized agent.
+```
+logs_enabled: true
+```
 
-To set the source for a given container filtered by image or label or name, update the log section in an integration or custom .yaml file in your agent's `conf.d` directory or the directory we just created for containerized installation.
+To collect logs from all your containers without any filtering, add the following at the end of `docker.d/conf.yaml` in your agent's `conf.d` directory:
 
-```yaml
-#Log section
+```
 logs:
-
-    # - type : (mandatory) type of log input source (tcp / udp / file / docker)
-    #   port / path : (mandatory) Set port if type is tcp or udp. Set path if type is file
-    #   service : (mandatory) name of the service owning the log
-    #   source : (mandatory) attribute that defines which integration is sending the logs
-    #   sourcecategory : (optional) Multiple value attribute. Can be used to refine the source attribtue
-    #   tags: (optional) add tags to each logs collected
-
-  - type: docker
-    image: <IMAGE_NAME>    #or label: <MY_LABEL> or name: <CONTAINER_NAME>
-    source: <SOURCE>
-    sourcecategory: <SOURCE_CATEGORY>
-    service: <SERVICE>
+    - type: docker
+      service: docker
+      source: docker
 ```
 
-When filtering on the container image, both exact container image name or short names are supported.
-If you have one container running `library/httpd:latest`, the following filtering matches this image name:
+[Restart the Agent](https://docs.datadoghq.com/agent/faq/agent-commands/?tab=agentv6#restart-the-agent) to see all your container logs in your platform.
+
+**Important note**: Integration Pipelines and Processors are not installed automatically as the source and service are set to the `docker` generic value.
+The source and service values can be overriden thanks to Autodiscovery as described below; it automatically installs integration Pipelines that parse your logs and extract all the relevant information from them.
 
-* - `image: httpd`
-* - `image: library/httpd`
-* - `image: httpd:latest`
-* - `image: library/httpd:latest`
+{{% /tab %}}
+{{< /tabs >}}
 
-For more examples of configuration files or Agent capabilities (such as filtering, redacting, multiline, ...) read [the advanced log collection functions][7].
+### Activate Log Integrations
 
-#### Option 2: Autodiscovery
+The second step is to use Autodiscovery to customize the `source` and `service` value. This allows Datadog to identify the log source for each container.
 
-Since version 6.2 of the Datadog Agent, you can configure log collection directly in the container labels.
+Since version 6.2 of the Datadog Agent, you can configure log collection directly in the container labels. 
```

Comment:
Pod annotations work only since `6.5`

Reply:
Yeah we should remove reference to specific version (container labels do work since 6.2 though)